cli/pcluster/configure/easyconfig.py

```diff
@@ -143,8 +143,6 @@ def configure(args):
         # Set provided region into os environment for suggestions and validations from here on
         if os.environ["AWS_DEFAULT_REGION"] != aws_region_name:
             os.environ["AWS_DEFAULT_REGION"] = aws_region_name
```

Comment:
minor: this conditional statement could be simplified, because we are not reading the config file again.
Only `os.environ["AWS_DEFAULT_REGION"] = aws_region_name` would work

Reply:
Done, thanks!